modules/ocf_irc/files/znc.service

```diff
@@ -0,0 +1,11 @@
+[Service]
+ExecStart=/usr/bin/znc --foreground --datadir /opt/znc
```

Comment:
data probably belongs in `/var/lib/znc` or `/srv/znc` and not `/opt/znc`?

quoth `man hier`:

> `/opt   This directory should contain add-on packages that contain static files.`

> `/var/lib/<package>    State data for packages and subsystems (optional).`

> `/srv   This directory contains site-specific data that is served by this system.`